man/man4/zfs.4

```diff
@@ -1848,6 +1848,16 @@ This setting will not reduce the write size below a single block.
 Capped at a maximum of
 .Sy 32MB .
 .
+.It Sy zfs_recv_best_effort_corrective Ns = Ns Sy 0 Ns Pq int
+When this variable is set to non-zero a corrective receive:
+  1) Does not enforce the restriction of source & destination snapshot GUIDs
+  matching.
+  2) If there is an error during healing, the healing receive is not
+  terminated instead it moves on to the next record.
+.
+Default value is
+.Sy 0 .
```

Comment:
(a) make this a list, (b) drop the default value sentence, (c) drop the Ns before Pq